src/testing/adaptive_sampling_testing.cpp

```diff
@@ -0,0 +1,143 @@
+        Parameters::AllParameters params = reinitParams(parameter);
+
+        std::unique_ptr<FlowSolver::FlowSolver<dim,nstate>> flow_solver_implicit = FlowSolver::FlowSolverFactory<dim,nstate>::select_flow_case(&params, parameter_handler);
+
+        auto functional_implicit = FunctionalFactory<dim,nstate,double>::create_Functional(params.functional_param, flow_solver_implicit->dg);
+
+        std::unique_ptr<FlowSolver::FlowSolver<dim,nstate>> flow_solver = FlowSolver::FlowSolverFactory<dim,nstate>::select_flow_case(&params, parameter_handler);
+        auto ode_solver_type = Parameters::ODESolverParam::ODESolverEnum::pod_petrov_galerkin_solver;
+        std::shared_ptr<ProperOrthogonalDecomposition::OfflinePOD<dim>> pod_standard = std::make_shared<ProperOrthogonalDecomposition::OfflinePOD<dim>>(flow_solver->dg);
+        flow_solver->ode_solver =  PHiLiP::ODE::ODESolverFactory<dim, double>::create_ODESolver_manual(ode_solver_type, flow_solver->dg, pod_standard);
+        flow_solver->ode_solver->allocate_ode_system();
+        auto functional = FunctionalFactory<dim,nstate,double>::create_Functional(params.functional_param, flow_solver->dg);
+
+        flow_solver_implicit->ode_solver->steady_state();
+        flow_solver->ode_solver->steady_state();
+
+        dealii::LinearAlgebra::distributed::Vector<double> standard_solution(flow_solver->dg->solution);
+        dealii::LinearAlgebra::distributed::Vector<double> implicit_solution(flow_solver_implicit->dg->solution);
+
+        double standard_error = ((standard_solution-=implicit_solution).l2_norm()/implicit_solution.l2_norm());
+        double standard_func_error = functional->evaluate_functional(false,false) - functional_implicit->evaluate_functional(false,false);
+
+        pcout << "State error: " << std::setprecision(15) << standard_error << std::setprecision(6) << std::endl;
+        pcout << "Functional error: " << std::setprecision(15) << standard_func_error << std::setprecision(6) << std::endl;
+
+        data_table->add_value("Parameter 1", parameter(0));
+        data_table->add_value("Parameter 2", parameter(1));
+        data_table->add_value("FOM func", functional_implicit->evaluate_functional(false,false));
+        data_table->add_value("ROM func", functional->evaluate_functional(false,false));
+        data_table->add_value("Func error", standard_func_error);
+        data_table->add_value("State error", standard_error);
+
+        data_table->set_precision("Parameter 1", 16);
+        data_table->set_precision("Parameter 2", 16);
+        data_table->set_precision("FOM func", 16);
+        data_table->set_precision("ROM func", 16);
+        data_table->set_precision("Func error", 16);
+        data_table->set_precision("State error", 16);
+
+        std::ofstream data_table_file("adaptation_data_table.txt");
+        data_table->write_text(data_table_file, dealii::TableHandler::TextOutputFormat::org_mode_table);
+    }
+    return 0;
```

Comment:
What is this test checking for (and since this is the only return statement, does this test always pass)?

Reply:
It's a test used for validation testing and producing additional results, it does not actually test anything. I have commented out the tests in the CMakeLists for this test.